DrawerController/DrawerController.swift

```diff
@@ -225,10 +225,32 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
     }
     
     /// Actions
+    @IBAction func openLeftSide(_ sender: Any) {
+        self.openSide(.left)
```

Comment:
I modified it to not use `self` when accessing inside from inside a class.
Please fix it. (and openRightSide method)

```swift
openSide(.left)
```

Reply:
Well, it is better practice to always use `self.`, but sure.

Reply:
Updated.